### PR TITLE
OCM-2237: Add support for adding path to account roles in creation

### PR DESCRIPTION
**Pull request by nirarg**
Also added option to seth the path in the managed and unmanaged clusters in order to allow using it from the clusters

### CODE REVIEW

**Comment by nirarg** [comment]
/hold 
This PR can't be merge before merging https://github.com/terraform-redhat/terraform-aws-rosa-sts/pull/17 and creating release for the modules (version 0.0.6)

**Review by osherdp — APPROVED** [review]

**Pull request merged**
